### PR TITLE
SNMP fingerprints for ScreenOS

### PR DESCRIPTION
**Pull request by jkennedy-r7**
I found these banners on Shodan.

### CODE REVIEW

File: xml/snmp_sysdescr.xml [hunk 1]
```diff
@@ -3397,7 +3397,21 @@ Copyright (c) 1995-2005 by Cisco Systems
...
+    <example os.version="5.4.0r3a.0">NetScreen-5GT version 5.4.0r3a.0 (SN: 0064092007003281, Firewall+VPN)</example>
+    <example os.version="5.4.0r2.0">NetScreen-5GT version 5.4.0r2.0 (SN: 0064092005010968, Firewall+VPN)</example>
+    <example os.version="6.2.0r2.0">NetScreen-5GT version 6.2.0r2.0 (SN: 0064122004003146, Firewall+VPN)</example>
+    <example os.version="5.4.0r20.0">NetScreen-208 version 5.4.0r20.0 (SN: 0099052005000121, Firewall+VPN)</example>
+    <param pos="0" name="os.vendor" value="Juniper"/>
+    <param pos="0" name="os.device" value="Firewall"/>
+    <param pos="0" name="os.product" value="ScreenOS"/>
+    <param pos="1" name="os.version"/>
+    <param pos="0" name="os.family" value="ScreenOS"/>
+    <param pos="0" name="os.family" value="ScreenOS"/>
```
Comment (jhart-r7) [inline xml/snmp_sysdescr.xml]: Duplicate.

File: xml/snmp_sysdescr.xml [hunk 2]
```diff
@@ -3397,7 +3397,21 @@ Copyright (c) 1995-2005 by Cisco Systems
   <!--======================================================================
                               Juniper
    =======================================================================-->
-  <fingerprint pattern="^Juniper (\S+) : (\d+\S+)$">
+  <fingerprint pattern="^NetScreen-[\w]+ version ([\w.]+) \(SN: [\d]+, Firewall\+VPN\)$">
```
Comment (jhart-r7) [inline xml/snmp_sysdescr.xml]: IMO, you should capture the model and assert it as `hw.model`.  This is done for one other thing in recog, and it just so happens to be related -- Juniper fingerprints via SNMP `sysDesc`

**Comment by jhart-r7** [comment]
LGTM.  I'll land this shortly.

**Comment by mdobrska-r7** [comment]
:shipit: !

**Pull request merged**